### PR TITLE
Feb 2018 Ethnologue files update

### PR DESCRIPTION
**Pull request by glasseyes**
and add extra country codes used by them

---
This change is [Reviewable](https://reviewable.io/reviews/sillsdev/libpalaso/665)

### CODE REVIEW

**Comment by glasseyes** [comment]
ready for review. win32 2.6 build succeeded on 2nd attempt

File: Palaso/Resources/CountryCodes.txt [hunk 1]
```diff
@@ -44,13 +48,16 @@ CL	Chile	Americas
 CM	Cameroon	Africa
 CN	China	Asia
 CO	Colombia	Americas
+CP	Clipperton Island
 CR	Costa Rica	Americas
 CU	Cuba	Americas
 CV	Cape Verde Islands	Africa
 CW	Curacao	Americas
 CY	Cyprus	Asia
+CX	 Christmas Island	Pacific
```
Comment (ermshiperete) [inline Palaso/Resources/CountryCodes.txt]: Probably doesn't matter, but there's an extra space in front of the name.

Comment (ermshiperete) [reply Palaso/Resources/CountryCodes.txt]: ...and this file seems to be sorted, so this line should come before CY.

File: Palaso/Resources/CountryCodes.txt [hunk 2]
```diff
@@ -44,13 +48,16 @@ CL	Chile	Americas
 CM	Cameroon	Africa
 CN	China	Asia
 CO	Colombia	Americas
+CP	Clipperton Island
 CR	Costa Rica	Americas
 CU	Cuba	Americas
 CV	Cape Verde Islands	Africa
 CW	Curacao	Americas
 CY	Cyprus	Asia
+CX	Christmas Island	Pacific
```
Comment (ermshiperete) [inline Palaso/Resources/CountryCodes.txt]: Just noticed that this file is sorted, so this line should probably come before CY.

**Comment by glasseyes** [comment]
Review status: 0 of 3 files reviewed at latest revision, 2 unresolved discussions.

---

*[Palaso/Resources/CountryCodes.txt, line 57 at r1](https://reviewable.io:443/reviews/sillsdev/libpalaso/665#-L86OyXb24E0dl2zKzrg-r1-57:-L86a4GXc5qU1HZHFyGN:b-896fix) ([raw file](https://github.com/sillsdev/libpalaso/blob/b986a10b70bda9728686dcabde1faf83033ea310/Palaso/Resources/CountryCodes.txt#L57)):*
<details><summary><i>Previously, ermshiperete (Eberhard Beilharz) wrote…</i></summary><blockquote>

...and this file seems to be sorted, so this line should come before CY.
</blockquote></details>

Done.

---

*[Palaso/Resources/CountryCodes.txt, line 57 at r2](https://reviewable.io:443/reviews/sillsdev/libpalaso/665#-L86OyXb24E0dl2zKzrg-r2-57:-L86a4p-tXHG23ZPgDiP:b-896fix) ([raw file](https://github.com/sillsdev/libpalaso/blob/8f6bc3bdd7edd5c4841fc7d01ce6901ff808eb52/Palaso/Resources/CountryCodes.txt#L57)):*
<details><summary><i>Previously, ermshiperete (Eberhard Beilharz) wrote…</i></summary><blockquote>

Just noticed that this file is sorted, so this line should probably come before CY.
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/sillsdev/libpalaso/665)*

**Comment by glasseyes** [comment]
eberhard couldn't get to reviewable for some reason to acknoweledge the fixes but approved on github and verbally

**Pull request merged**